lint/mypy.bzl

```diff
@@ -0,0 +1,231 @@
+"""API for declaring a mypy lint aspect that visits py_library rules.
+
+Typical usage:
+
+```
+load("@aspect_rules_lint//lint:mypy.bzl", "mypy_aspect")
+
+mypy = mypy_aspect(
+    binary = "@@//:mypy",
+    configs = "@@//:pyproject.toml",
+)
+```
+
+Optional: enable it in `.bazelrc`:
+
+```
+# Add mypy type-check validation actions to py_library targets.
+# Disable for a particular build with --norun_validations
+build --aspects=//tools:lint.bzl%mypy
+```
+
+### Reporting
+
+Unlike most linters hosted in rules_lint, mypy produces only error semantics.
+That means that typecheck violations will input_depsets in failed build actions, rather than
```

Comment:
I'm not sure I can make sense of this statement, "will input_depsets"?

Reply:
oops, was an over-eager searchreplace